x-pack/platform/plugins/shared/index_management/public/application/mount_management_section.ts

```diff
@@ -129,7 +130,7 @@ export async function mountManagementSection({
 }: {
   coreSetup: CoreSetup<StartDependencies>;
   usageCollection: UsageCollectionSetup;
-  params: ManagementAppMountParams;
+  params: ManagementAppMountParams | SearchIndicesAppMountParams;
```

Comment:
I left another similar comment before, got a bit confused and deleted it BUT - I'm confused why there needs to be different types here. It seems like they should be the same or at least extremely similar.

Reply:
`SearchIndicesAppMountParams` is a subset of `ManagementAppMountParams`.  The [ManagementAppMountParams](https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/management/public/types.ts#L76) has few required field which would be undefined when mounting from `search_indices` plugin.  This would cause type_check error. 
Also from the code, I understand those fields are not used by [mount_management_section](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/index_management/public/application/mount_management_section.ts#L140) in index management.

Reply:
I think other way to simplify,  is to create a new type `IndexManagementAppMountParams` which is a subset of [ManagementAppMountParams](https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/management/public/types.ts#L76) and replace [mount_management_section](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/index_management/public/application/mount_management_section.ts#L140) to use this type instead of ManagementAppMountParams. I will try that and confirm nothing breaks and update this PR